Upstream: Yes
Reason: Needed for multiarch fixes

From bfa0152e77e9c22fd1b8ca7ea105a961061faee6 Mon Sep 17 00:00:00 2001
From: Chandler Carruth <chandlerc@gmail.com>
Date: Sat, 14 Feb 2015 09:05:56 +0000
Subject: [PATCH 1/3] [lit] Make the gold plugin support testing work with a
 python3 interpreter. Seems that's a better path than pinning to python2.7.

Thanks to Justin for prodding me toward a fix. =]

git-svn-id: https://llvm.org/svn/llvm-project/llvm/trunk@229247 91177308-0d34-0410-b5e6-96231b3b80d8
---
 test/lit.cfg | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/lit.cfg b/test/lit.cfg
index cd615b3..3023b93 100644
--- a/test/lit.cfg
+++ b/test/lit.cfg
@@ -349,7 +349,7 @@ def have_ld_plugin_support():
         return False
 
     ld_version = subprocess.Popen(['ld', '--version'], stdout = subprocess.PIPE)
-    if not 'GNU gold' in ld_version.stdout.read():
+    if not 'GNU gold' in ld_version.stdout.read().decode():
         return False
     ld_version.wait()
 
-- 
2.3.3

